src/d_soundfile.c

```diff
@@ -19,7 +19,9 @@ objects use Posix-like threads. */
 #include <stdio.h>
 #include <pthread.h>
 
-/* Supported sample formats: LPCM (16 or 24 bit int) & 32 bit float */
+/* Supported sample formats: LPCM (16 or 24 bit int) & 32 or 64 bit float */
+
+#define VALID_BYTESPERSAMPLE(b) (b == 2 || b == 3 || b == 4 || b == 8)
```

Comment:
prefer putting parens around the macro vars:

```
#define VALID_BYTESPERSAMPLE(b) ((b) == 2 || (b) == 3 || (b) == 4 || (b) == 8)
```

Reply:
Out of curiosity, why?

Reply:
done in latest commit